test/functional/tool_wallet.py

```diff
@@ -188,6 +188,8 @@ def test_invalid_tool_commands_and_args(self):
         self.assert_raises_tool_error('Invalid command: help', 'help')
         self.assert_raises_tool_error('Error: two methods provided (info and create). Only one method should be provided.', 'info', 'create')
         self.assert_raises_tool_error('Error parsing command line arguments: Invalid parameter -foo', '-foo')
+        self.assert_raises_tool_error('No method provided. Run `bitcoin-wallet -help` for valid methods.')
```

Comment:
in fa06bce4ac17f93decd4ee38c956e7aa55983f0d:

nit: Commit message could have been a bit more descriptive ;)

Reply:
Thanks, will change on the next push